plugins/channel_header_plug/channel_header_plug.tsx

```diff
@@ -123,9 +127,22 @@ class ChannelHeaderPlug extends React.PureComponent<ChannelHeaderPlugProps, Chan
         super(props);
         this.state = {
             dropdownOpen: false,
+            disableButtonsClosingRHS: false,
         };
     }
 
+    componentDidUpdate(prevProps: ChannelHeaderPlugProps) {
+        if (prevProps.sidebarOpen && !this.props.sidebarOpen) {
+            // eslint-disable-next-line react/no-did-update-set-state
+            this.setState({disableButtonsClosingRHS: true});
```

Comment:
To avoid the extra renders caused by storing this in state and fix that warning, I think we could store `disableButtonsClosingRHS` as a member variable. It doesn't affect rendering anyway, so it doesn't need to be in state.